frontend/src/ClassView/StudentAssignmentCard.tsx

```diff
@@ -7,6 +7,10 @@ import Card from "react-bootstrap/Card";
 import ProgressBar from "react-bootstrap/ProgressBar";
 
 import { Link } from "react-router-dom";
+import {
+  buildStyles,
+  CircularProgressbarWithChildren,
+} from "react-circular-progressbar";
```

Comment:
Could what you're looking for be [`variant="border"`](https://react-bootstrap.github.io/components/spinners/)?